ios-nocodesign-12-1-dep-9-0-bitcode-cxx11.cmake

```diff
@@ -0,0 +1,44 @@
+# Copyright (c) 2017, Ruslan Baratov
+# All rights reserved.
+ 
```

Comment:
Please remove trailing whitespaces.

Reply:
I don't see any trailing whitespaces. Please elaborate.

Reply:
> I don't **see** any trailing whitespaces

You can't see them, they are whitespaces :)

> Please elaborate

Check this commit: https://github.com/ruslo/polly/pull/283/commits/f114519882896736e8edc96910f494834a792547